tests/suites/ck/ck.sh

```diff
@@ -3,14 +3,31 @@
 run_deploy_ck() {
 	echo
 
-	local name model_name file overlay_path kube_home
+	local name model_name file overlay_path overlay_url kube_home
 	name="deploy-ck"
 	model_name="${name}"
 	file="${TEST_DIR}/${model_name}.log"
 
 	ensure "${model_name}" "${file}"
 
-	overlay_path="./tests/suites/ck/overlay/${BOOTSTRAP_PROVIDER}.yaml"
+	case "${BOOTSTRAP_PROVIDER:-}" in
+	"ec2")
+		overlay_url="https://raw.githubusercontent.com/charmed-kubernetes/bundle/main/overlays/aws-storage-overlay.yaml"
```

Comment:
Do we want to test their main branch instead of the stable release?
If so, we need to change the below deploy arg.

Reply:
there isn't overlays for stable vs edge, main branch on the bundle repo appears to be the best we can get.